samples/SunburstChartExample/sunburstppt.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.IO.Packaging" Version="4.7.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="DocumentFormat.OpenXml">
+      <HintPath>..\..\..\forks\Open-XML-SDK\bin\Debug\DocumentFormat.OpenXml\netstandard2.0\DocumentFormat.OpenXml.dll</HintPath>
```

Comment:
This hint path will not work - looks like  aprivate build

Reply:
changed to project reference after adding to SDK solution